Library/Homebrew/extend/ENV/super.rb

```diff
@@ -39,6 +39,14 @@ def setup_build_environment(formula = nil)
     super
     send(compiler)
 
+    if compiler == :llvm_clang && Formula["llvm"].installed?
+      FileUtils::mkdir "#{HOMEBREW_TEMP.to_s}/brew-llvm-openmp"
+      FileUtils::ln_sf "#{Formula["llvm"].lib}/clang/#{Formula["llvm"].version.to_s}/include/omp.h",
+                       "#{HOMEBREW_TEMP.to_s}/brew-llvm-openmp/omp.h"
+      FileUtils::ln_sf "#{Formula["llvm"].lib}/libomp.dylib",
+                       "#{HOMEBREW_TEMP.to_s}/brew-llvm-openmp/libomp.dylib"
```

Comment:
I was concerned that that would cause libc++ and friends to be pulled in instead of the system versions